src/components/LoginForm.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import { bool, array, func } from 'prop-types';
+import { faSpinner } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+class LoginForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: '',
+      password: ''
+    };
+  }
+
+  handleInputChange = (event) => {
+    const { target: { name, value } } = event;
+
+    this.setState({
+      [name]: value
+    });
+  }
+
+  handleSubmit = (event) => {
+    this.props.onSubmit(this.state);
+    event.preventDefault();
+  }
+
+  render() {
+    const { isLoading, errors } = this.props;
+
+    return (
+      <form className="login-form" onSubmit={this.handleSubmit}>
+        <label className="login-form-label" htmlFor="email">EMAIL</label>
+        <input
```

Comment:
Maybe create a common component called Input because you are probably going to re use it a lot